plugins/GSdx/Renderers/HW/GSRendererHW.cpp

```diff
@@ -1714,6 +1720,26 @@ bool GSRendererHW::OI_BigMuthaTruckers(GSTexture* rt, GSTexture* ds, GSTextureCa
 	return true;
 }
 
+bool GSRendererHW::OI_DBZBT2(GSTexture* rt, GSTexture* ds, GSTextureCache::Source* t)
+{
+	// Sprite rendering
+	if (m_context->FRAME.PSM != PSM_PSMCT32)
+		return true;
+	if (PRIM->TME && m_context->TEX0.PSM != PSM_PSMCT32)
+		return true;
+	GSVector4i r = m_r;
+	if (!(r == GSVector4i(0, 0, 64, 64)).alltrue())  // Rendering region is 64x64
```

Comment:
Any reason you can't just use `m_r`?

Reply:
No, it's just a leftover from previous code where the `r` was recomputed from scratch instead of reusing `m_r`